src/test/java/net/sf/jabref/importer/fileformat/PdfContentImporterTest.java

```diff
@@ -17,4 +31,19 @@ public void doesNotHandleEncryptedPdfs() throws IOException {
 
         assertEquals(Collections.emptyList(), result);
     }
-}
\ No newline at end of file
+
+    @Test
+    public void correctContent() throws IOException {
+        PdfContentImporter importer = new PdfContentImporter();
+        // The test folder contains pairs of PDFs and BibTeX files. We check each pair.
+        // Currently only one pair available
+        List<String> prefixes = Arrays.asList("LNCS-minimal");
+        for (String prefix : prefixes) {
+            try (InputStream is = PdfContentImporter.class.getResourceAsStream(prefix + ".pdf")) {
+                List<BibEntry> result = importer.importEntries(is, null);
+                Assert.assertEquals(1, result.size());
+                BibtexEntryAssert.assertEquals(PdfContentImporterTest.class, prefix + ".bib", result.get(0));
```

Comment:
Yes, I think so. :innocent:. Think, it is easier for writing tests and debugging. The should-be-data is more close to the thing the user sees, but also close to the data generated by the importer.